pages/news.vue

```diff
@@ -1,32 +1,30 @@
 <template>
   <v-container>
+    <h1 class="mt-3"> 
+      {{ $t('news.title') }} 
+    </h1>
+    <p class="mb-6"> 
+      {{ $t('news.blurb') }} 
+    </p>
+    <div v-if="!loaded">
+      <v-skeleton-loader type="card-avatar, article" elevation="2" class="mt-9" height="400" /> 
+      <v-skeleton-loader type="card-avatar, article" elevation="2" class="my-8" height="400" /> 
+      <v-skeleton-loader type="card-avatar, article" elevation="2" class="my-8" height="400" /> 
+    </div>
     <v-row>
       <v-col v-if="loaded">
-        <h1>{{ $t('news.title') }}</h1>
-        <p>{{ $t('news.blurb') }}</p>
-        <v-card v-for="(newsItem, i) in newsItems.json.features" :key="i" class="my-6">
-          <v-card-title v-if="en" class="info">
-            {{ newsItem.properties.title_en }}
+        <v-card v-for="(newsItem, i) in newsItems.json.features" :key="i" class="mb-6">
+          <v-card-title class="info">
+            {{ newsItem.properties[`title_${$i18n.locale}`] }}
           </v-card-title>
-          <v-card-title v-if="fr" class="info">
-            {{ newsItem.properties.title_fr }}
-          </v-card-title>
-          <v-card-subtitle v-if="en" class="info">
-            <span class="blue--text text--darken-3">{{ newsItem.properties.published.slice(0,11) }}</span>
-            <v-chip v-for="(keyword, j) in newsItem.properties.tags_en.split(',')" :key="j" class="ma-2" small> 
-              <span style="font-size: 12px">{{ keyword }}</span> 
-            </v-chip>
-          </v-card-subtitle>
-          <v-card-subtitle v-if="fr" class="info">
+          <v-card-subtitle class="info">
             <span class="blue--text text--darken-3">{{ newsItem.properties.published.slice(0,11) }}</span>
-            <v-chip v-for="(keyword, j) in newsItem.properties.tags_fr.split(',')" :key="j" class="ma-2" small>
+            <v-chip v-for="(keyword, j) in newsItem.properties[`tags_${$i18n.locale}`].split(',')" :key="j" class="ma-2" small>
               <span style="font-size: 12px">{{ keyword }}</span>
```

Comment:
let's remove this `style` or `<span>` to avoid use of absolute font-size. `<v-chip>` has `small` and `x-small` properties to use instead.